modules/ROOT/pages/get-started.adoc

```diff
@@ -0,0 +1,143 @@
+. <<monitor-transmissions,Monitor transmissions and individual messages>>.
+. <<troubleshoot-acks,Troubleshoot rejected or overdue acknowledgements>>.
+
+[[assign-permissions]]
+== Assign Permissions to Partner Manager Users
+
+For each sandbox and production environment, use Anypoint Access Management to assign permissions to each Partner Manager user. You can assign the following permissions:
+
+*  *Partner Manager Administrator* 
++
+** Has full permissions
+* *View Host, Partners, and Message Flows* 
++
+** Has view-only access to the host, partner, and message flow configurations
+** Cannot view transaction activity
+* *Manage Partners and Message Flows* 
++
+** Can create, modify, and delete partners or message flow configurations
+** Can view host partner configurations
+** Cannot view and manage transaction activity
+* *Manage Activity* 
++
+** Can view and manage transaction activity
+** Cannot view or modify either partner or message flow configurations
+* *Manage Host* 
++
+** Can create, modify, and delete host configurations
+** Cannot view or modify trading partner configurations or transaction activity 
+
++
+This access applies even if the user has the Anypoint Platform Org Admin role.
+* *View Activity* 
++
+** Has view-only access to transaction activity
+** Cannot view or modify either partner or message flow configurations
+
+For more information, see xref:access-management::permissions-by-product.adoc[] .
+
+[[set-up-env]]
+== Set Up the Partner Manager Environment
+
+You can deploy Partner Manager to CloudHub or you can use Partner Manager on-premises:
+
+* To deploy Partner Manager to CloudHub, use the *Partner Manager Settings* page in Anypoint Access Management to specify CloudHub deployment options for the Partner Manager sandbox and production environments.
++
+For more information, see xref:cloudhub-deploy-options.adoc[].
++
+* To use Partner Manager on-premises:
+. Install and configure Mule runtime engine (Mule) on-premises.
+. Use Anypoint Runtime Manager to register Partner Manager in the appropriate environment.
+. Create a server group to which to add the servers you create.
+
++
+For more information, see xref:deploy-onpremise.adoc[].
+
+[[payload-storage-api]]
+== Set Up the Payload Storage API
+
+Partner Manager requires that you deploy a payload storage API for storing transmission information. You can connect this API to your choice of a storage location, such as a database, Amazon S3, or a file system.
+
+For more information, see xref:setup-payload-storage-API.adoc[].
+
+[[configure-host-profile]]
+== Configure a Host Profile
+
+Use the <*host*> page in Partner Manager to configure a host profile for your Anypoint Platform organization or business group. A host  profile consists of the EDI and AS2 identfiers used by your organization, associated digital certificates, Source at Host and Target at Host endpoints, message types, Payload Storage API details, and global settings for EDIFACT receive messages.
```

Comment:
See previous. Also, curious about "<*host*>" markup. Correct?

Reply:
Are all these init caps for proper nouns? If not, lc. If UI elements, then use bold?

Reply:
<*Host*> is correct because this is a variable.